manticore/__main__.py

```diff
@@ -79,51 +78,38 @@ def parse_arguments():
 def main():
     args = parse_arguments()
 
-    m = Manticore(args.argv[0], args.argv[1:], disasm=args.disasm)
-    m.policy = args.policy
-    m.args = args
+    log.set_verbosity(args.v)
+
+
+    #TODO 
+    initial_state = make_initial_state(args.argv[0], argv=args.argv[1:],  disasm=args.disasm)
+    m = Manticore(initial_state, workspace_url=args.workspace,  policy=args.policy)
```

Comment:
should policy be an argument to m.run()?

Reply:
It sounds like it but it's _currently_? needed at executor constructor.

Reply:
ok, we can address later